extensions/wikia/AchievementsII/services/AchImageUploadService.class.php

```diff
@@ -23,71 +23,71 @@ private static function imagecopymerge_alpha($dst_im, $src_im, $dst_x, $dst_y, $
 		// Find the most opaque pixel in the image (the one with the smallest alpha value)
 		$minalpha = 127;
 		for( $x = 0; $x < $w; $x++ )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS121)